src/spec.ts

```diff
@@ -20,7 +20,7 @@ import {contains, Dict, duplicate, hash, vals} from './util';
 
 export type TopLevel<S extends BaseSpec> = S & TopLevelProperties & {
   /**
-   * URL to JSON schema for this Vega-Lite specification.
+   * URL to [JSON schema](http://json-schema.org/) for this Vega-Lite specification. Unless you have a reason to change this, use `https://vega.github.io/schema/vega-lite/v2.json`. Setting the `$schema` property allows automatic validation and autocomplete in editors that support JSON schema.
```

Comment:
"for a Vega-Lite specification"?

(This => a)

Reply:
I think this is correct here as you can have your own dialect of Vega-Lite or use a specific version.

Reply:
This will appear in the docs, and it will be weird which specification is "this"?

Reply:
Done